internal/stringify/fmt.go

```diff
@@ -104,3 +104,11 @@ func Normalize(kvs ...any) map[string]any {
 
 	return norm
 }
+
+func NormalizeMap[K comparable, V any](m map[K]V) map[string]any {
+	kvs := make([]any, 0, len(m)*2)
+	for k, v := range m {
+		kvs = append(kvs, k, v)
+	}
+	return Normalize(kvs...)
```

Comment:
nit: i need to add a linter for this.
```suggestion
	kvs := make([]any, 0, len(m)*2)

		for k, v := range m {
		kvs = append(kvs, k, v)
	}

	return Normalize(kvs...)
```